packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -107,6 +108,13 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
   }
 
   Future<void> _toggleBreakpoint(ScriptRef script, int line) async {
+    // The VM doesn't support debugging for system isolates and will crash on
+    // a failed assert in debug mode. Disable the toggle breakpoint
+    // functionality for system isolates.
```

Comment:
in this case, we should also make sure we aren't drawing the little gray dots on executable lines in the gutter for system isolate scripts.

Reply:
Done. Also went ahead and forced the mouse cursor to the default instead of the pointing hand when there's nothing to interact with in the gutter.